docker-compose.yaml

```diff
@@ -1,34 +1,32 @@
 version: '3.7'
+# each service runs a container needed to run ceqr-app
 services:
-  # ceqr app service
+  # rails container
   backend:
-    build: backend
-    entrypoint: ./entrypoint.sh
+    build: backend # build the image for the container from backend/Dockerfile
+    entrypoint: ./entrypoint.sh # executes backend/entrypoint.sh on container startup
     volumes:
-      - ./backend:/app
+      # HOST:CONTAINER file mapping
+      - ./backend:/app # map our rails code into the container's /app directory
     ports:
-      - "3000:${SERVICE_PORT}"
-    environment:
-      PORT: ${SERVICE_PORT}
-      DATABASE_URL: postgis://postgres@postgis:5432/ceqr_rails
-      CEQR_DATA_DB_URL: postgis://doadmin:${PSQL_PASSWORD}@ceqr-data-readonly-do-user-1939427-0.db.ondigitalocean.com:25060/ceqr_data
-      JWT_SALT: saltysaltysalt
-      SENDGRID_KEY: ''
-      ADMIN_EMAILS: 'test@test.com'
+      # HOST:CONTAINER port mapping
+      - "3000:3000" # map port 3000 on our machine to the container's port 3000 so we (our our frontend container) can go to localhost:3000 to access our container's rails server
```

Comment:
nit: extra `our`

Reply:
thanks for spotting!  Will fix that in the next update to the doc